src/transformers/modeling_albert.py

```diff
@@ -652,6 +656,14 @@ def forward(
 
         """
 
+        if "masked_lm_labels" in kwargs:
+            warnings.warn(
+                "The `masked_lm_labels` argument is deprecated and will be removed in a future version, use `labels` instead.",
+                DeprecationWarning,
+            )
+            labels = kwargs.pop("masked_lm_labels")
+        assert kwargs == {}, "Unexpected keyword argument."
```

Comment:
Oh, I can use f-strings? I thought the minimum version of python was 3.5 for the project (and f-strings are in 3.6).

Reply:
We're on python 3.6+ now, since https://github.com/huggingface/transformers/commit/2187c49f5cde57306c3fd1eb67dbc68fab9c6403!

Reply:
Nice! Will add that in all the kwargs assert then.

Reply:
> Made the same for all models since @julien-c liked it. A few comments as I was reading and deprecating.
> 
> * I found a few more wrong docstrings (mentioning `lm_label` when the arg was called `labels`) so fixed them.
> * As @patrickvonplaten mentioned on #4711, `BertForMaskedLM` should be split in two (and add a `BertWithLMHead`) to remove the `lm_labels` argument. I made this a TODO to avoid this PR become too big.
> * The GPT2 and openai models also have a version with two labels (`GPT2DoubleHeadsModel` and `OpenAIDoubleHeadsModel`), I renamed `lm_labels` to `labels` there but there may still be a need for a second label. Can revert the change on those models if we want each labels arg to have a useful name.
> * In `LongformerModel`, the `label` argument is not used, should it be dropped?
> 
> Also, I note that quite a few docstrings have example that don't match the model they document (for instance an electra model was using Bert as an example, but there are a few instances).

The `masked_lm_labels` argument in `LongformerModel` forward function is definitely an error - it should be removed. Feel free to remove it here or I can also remove it in another PR.